src/Utils/OccRunner.php

```diff
@@ -55,7 +55,7 @@ public function run($command, $args = [], $asJson = false){
 			$cmdLine = trim($command . ' ' . $extra);
 			foreach ($args as $optionTitle => $optionValue){
 				if (strpos($optionTitle, '--') === 0){
-					$line = trim("$optionTitle $optionValue");
+					$line = trim("$optionTitle=$optionValue");
```

Comment:
@PVince yes, a few lines below. And the issue is caused by this escaping, that's why we need to get rid of space symbol here